subgraphs/aave-forks/src/helpers.ts

```diff
@@ -1059,14 +97,125 @@ export function getBorrowBalance(
 
   // get account's balance of stable debt
   if (market._sToken) {
-    const sTokenContract = AToken.bind(Address.fromString(market._sToken!));
+    const sTokenContract = AToken.bind(Address.fromBytes(market._sToken!));
     const trySDebtTokenBalance = sTokenContract.try_balanceOf(account);
     sDebtTokenBalance = trySDebtTokenBalance.reverted
       ? BIGINT_ZERO
       : trySDebtTokenBalance.value;
   }
 
-  const totalDebt = sDebtTokenBalance.plus(vDebtTokenBalance);
+  // const totalDebt = sDebtTokenBalance.plus(vDebtTokenBalance);
+
+  return [sDebtTokenBalance, vDebtTokenBalance];
+}
+
+export function getCollateralBalance(market: Market, account: Address): BigInt {
+  let collateralBalance = BIGINT_ZERO;
+  const aTokenContract = AToken.bind(Address.fromBytes(market.outputToken!));
+  const balanceResult = aTokenContract.try_balanceOf(account);
+  if (balanceResult.reverted) {
+    log.warning(
+      "[getCollateralBalance]failed to get aToken {} balance for {}",
+      [market.outputToken!.toHexString(), account.toHexString()]
+    );
+    return collateralBalance;
+  }
+
+  let exchangeRate = BIGDECIMAL_ONE;
```

Comment:
excahnge rate is always 1 in aave since all of the aTOkens and debt tokens are rebasing

Reply:
simplified.